commands.py

```diff
@@ -0,0 +1,22 @@
+import os
+import json
+def create(schima):
```

Comment:
There is two files named `command` and `commands` which is confusing. 
I would suggest `command.py` is the facade design pattern application. and commands should be `command`

Reply:
You should have interface `ICommand` and multiple classes. CreateCommand extends ICommand and so on